free-3.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I imagine this being helpful for novice potery writers who may need help writing poems in certain formats, such as sonnets and limericks, and want various types of help in doing so. Now, the experience may be difficult for writers who have a hard time with meter or with rhyming. I imagine this DSL to suggest words to users to help them with rhyme scheme, point out overused words, and look up pronounciations of words to make sure meter and rhythm is correct.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because poems, and especially structured poems, are a very particular and strictly defined subset of natural language, that can be checked as long as the rules are properly described. I think it addresses the need by acting as a domain-specific spellchecker, to which inputs (which type of poem, the poem itself) can be input, and suggestions can be output.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I like the idea because it seems like something that could be done in a fairly straightforward way. I came up with it just thinking of DSL ideas -- it seemed like something I could do and something that would be fun to play around with.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Poems with well defined structure
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I think the user would input to the language a specific poem type, as well as a text file or string which is the poem itself. I think a text file would be logical because then the program could look for line breaks more easily. The program would then output either a list of suggestions referencing line numbers and specific words, or maybe a text file of suggestions based on the poem type, or some similar format.
```

Comment:
Although focusing on a GUI too much can take away from the language design aspects, I think a GUI done the right way can be closely coupled with language design. _Sorry to be beating this point to death._